rules/macos/defense_evasion_tcc_bypass_mounted_apfs_access.toml

```
@@ -0,0 +1,43 @@
+[metadata]
+creation_date = "2020/01/04"
+maturity = "production"
+updated_date = "2020/01/04"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies the use of mount_apfs command to mount the entire file system through APFS snapshots as read-only and with
+the noowners flag, which enables an adversary accessing almost every file in the file system, including data of every
+user on the system as well as those protected by Apple’s privacy framework (TCC).
```

```suggestion
Identifies the use of the mount_apfs command to mount the entire file system through Apple File System (APFS) snapshots
as read-only and with the noowners flag set. This action enables the adversary to access almost any file in the file
system, including all user data and files protected by Apple’s privacy framework (TCC).
```

I tried to make this read a little easier.